package.json

```diff
@@ -78,7 +78,7 @@
     "tslint": "^5.11.0",
     "typescript": "^3.3.3",
     "winston": "^2.2.0",
-    "@ampretia/x509": "^0.4.0"
+    "@ampretia/x509": "0.4.0"
```

Comment:
Mark has made this ^0.4.5 in master

Reply:
Yeah 0.4.5 isn't working, but Matthew's got us a 0.4.7, so I've updated to use that